src/test/scala/temple/builder/project/ProjectBuilderTest.scala

```diff
@@ -9,33 +9,36 @@ class ProjectBuilderTest extends FlatSpec with Matchers {
 
   it should "correctly create a simple project using postgres as the default" in {
     val project = ProjectBuilder.build(ProjectBuilderTestData.simpleTemplefile)
-    project.files shouldBe Map(
-      File("templeuser-db", "init.sql") -> ProjectBuilderTestData.simpleTemplefilePostgresCreateOutput,
-      File("templeuser", "Dockerfile")  -> ProjectBuilderTestData.simpleTemplefileUsersDockerfile,
-    )
+    val expected = Map(
+        File("templeuser-db", "init.sql") -> ProjectBuilderTestData.simpleTemplefilePostgresCreateOutput,
+        File("templeuser", "Dockerfile")  -> ProjectBuilderTestData.simpleTemplefileUsersDockerfile,
+      ) ++ ProjectBuilderTestData.simpleTemplefileKubeScripts
```

Comment:
Hmm not sure I'm a fan of this, would prefer to explicitly list every file - with its contents being the thing in `TestData`